src/app/public/modules/select-field/select-field.component.ts

```diff
@@ -210,11 +210,9 @@ export class SkySelectFieldComponent implements ControlValueAccessor, OnDestroy
       return;
     }
 
-    if (value) {
```

Comment:
Here's the change. We weren't letting anything `undefined` through the CVA. Removing this `if` check allows consumers to programmatically change the value. However, we had to add one check for tokens in this case (see below).